<launch>

<!-- <arg name="bag" value="bag2" /> -->

<param name="odom_x_init" value="0.0"/>
<param name="odom_y_init" value="0.0"/>
<param name="odom_ang_init" value="0.0"/>

<node pkg="rosbag" type="play" name="rosbag_play" args="--clock $(find project1)/bags/$(arg bag).bag"/>
<node pkg="project1" type="vel_calc" name="vel_calc"/>
<node pkg="project1" type="odom" name="odom" output="screen"/>
<node pkg="project1" type="tf2_broadcaster" name="tf2_broadcaster"/>
<node pkg="project1" type="wheel_calc" name="wheel_calc"/>

</launch>